force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -139,17 +218,25 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String returnHelloWorld() {
         // Initialize the result String
+        String result = '';
 
         // Use a for loop to append 'Hello World!;' to the result string three times
-        /* 
-         for (){
+        for (Integer i = 0; i < 3; i++) {
+            result += 'Hello World!';
             
-         }
-        */
+            // If it is not the last iteration, add a semicolon
+            if (i < 3) {
+                result += '; ';
+            }
+        }
```

Comment:
Here, your code is not following the method criteria as outlined in the method instructions (see below). Also, you don't seem to be running your test methods. Test methods ensure your code is correct. Finally, be sure to turn off any AI assistance (eg, Copilot) so that it doesn't interfere with your learning (you want to make sure you're getting the most out of this course by doing the hard thinking yourself -- ha).
```
* Question 9
     * Generate String "Hello World!" three times, with each occurrence separated by a semicolon.
     * Example: returnHelloWorld() should return "Hello World!; Hello World!; Hello World!; "
```
If you don't know how to run the test methods, be sure to ask your instructor, your cohort in Slack, or [sign up for a 1-on-1](https://docs.google.com/spreadsheets/d/18pibSNUy5nt-p-KAHTfWfvpWVDVoeC2hHNGxEcuBdfI/edit?gid=0#gid=0).

Review the method instructions and rewrite your code to meet the requirements.